Makefile.bootfiles

```diff
@@ -20,6 +20,9 @@ IGVM_TOOL:=
 KERNEL_PATH:=
 TAR2EXT4_TOOL:=bin/cmd/tar2ext4
 
+ROOTFS_DEVICE:=/dev/sda
+HASH_DEVICE:=/dev/sdb
```

Comment:
In a subsequent PR (maybe I will do it) we need to make the v2056.vmgs scheme the same as the kernel.vmgs sceme and completely get rid of all the two vhd based configurations. Once that is done I'd like to find a way to provide v2056.vmgs and uvmtester in a testing package so we can do automated soak testing of the UVM startup part.